0/building-your-contract.md

```diff
@@ -1,44 +1,20 @@
 Building Your Contract
 ===
 
-The ink! CLI also generates a build script called `build.sh`:
-
-```bash
-#!/bin/bash
-set -e
-
-PROJNAME=flipper
-
-# cargo clean
-# rm Cargo.lock
-
-CARGO_INCREMENTAL=0 &&
-cargo build --release --features generate-api-description --target=wasm32-unknown-unknown --verbose
-wasm2wat -o target/$PROJNAME.wat target/wasm32-unknown-unknown/release/$PROJNAME.wasm
-cat target/$PROJNAME.wat | sed "s/(import \"env\" \"memory\" (memory (;0;) 2))/(import \"env\" \"memory\" (memory (;0;) 2 16))/" > target/$PROJNAME-fixed.wat
-wat2wasm -o target/$PROJNAME.wasm target/$PROJNAME-fixed.wat
-wasm-prune --exports call,deploy target/$PROJNAME.wasm target/$PROJNAME-pruned.wasm
-```
-
-This file will be used to compile your contract source code to WASM. You can see that it depends on the Wasm utilities we installed earlier.
-
 To compile the smart contract, we need to run it:
 
 ```bash
-./build.sh
+cargo contract build
 ```
 
 If all goes well, you should see a `target` folder being created with 5 relevant files corresponding to the steps in the script:
 
 ```
 target
-├── flipper-fixed.wat
-├── flipper-pruned.wasm
-├── flipper.wasm
-└── flipper.wat
+└── flipper.wasm
 ```
 
-The final, optimized `flipper-pruned.wasm` file is what we will actually deploy to our Substrate chain.
+The final, optimized `flipper.wasm` file is what we will actually deploy to our Substrate chain.
```

Comment:
is it still "optimized"

This sentence should probably change with the new instructions

Reply:
It is. You can get the full WASM file on the `target/wasm32-unknown-unknown/release` dir.
But it could be good idea to rephrase now that the user will only see one file on the main _target_ dir.

Reply:
Also we can clearly say that those Wasm files are heavily optimized also in regards to our own things like removing the custom section and running certain tools even after all Rust and LLVM based optimizations.